beebop/assignClusters.py

```diff
@@ -16,8 +16,15 @@ def hex_to_decimal(sketches_dict):
 
 def get_clusters(hashes_list, p_hash, fs, db_paths, args):
     """
-    assign clusterIDs to sketches
-    hashes_list: list of json objects stored json object of multiple sketches
+    Assign cluster numbers to samples using PopPUNK.
```

Comment:
Somewhere, can we have a glossary about things like clusters and components? Could be done in another PR or here, a md file at the root would do